announcements/_posts/2019-09-06-w02-everthing-ssh.md

```diff
@@ -0,0 +1,20 @@
+---
+author: Jeffery Russell (@jrtechs)
+title: "Week #2: Everything SSH!"
+layout: post-event
+date-start: "2019-09-06 16:30"
+date-end: "2019-09-06 18:00"
+location: "GOL-2620 (Med. DB Lab)"
+---
+
+Hey Ritluggers,
+
+This week we got a very excitting meeting: we are diving deep into SSH.
+We are going to cover everything from basic SSH commands to some of the more advance topics.
+You need no background knowledge of SSH for this meeting, just show up and learn with us.
+
+After a quick presentation there will be a SSH CTF (capture the flag) esk challenge.
```

Comment:
The *esk* isn't the correct usage. I know what you're trying to say here, but I don't think that's the proper usage in a writing context.

Reply:
how about:
After a quick presentation, there will be a SSH CTF-esk (capture the flag) challenge.